### ISSUE TITLE
max amplitude ratios for data and syn

### ISSUE THREAD

**Issue by carltape** [adjtomo/pysep#116]
A common occurrence when plotting data and synthetics is that there is a unit scaling issue, such that when the seismograms are not normalized, one set appears to be flat-lined. Listing the max (absolute?) value of the filtered seismogram is very helpful, as done [here](https://github.com/adjtomo/pysep/blob/master/pysep/recsec.py#L1525). It would be great if the two amplitudes could be used to show the ratio as A2/A1 and also ln( A2 / A1 ) -- by scanning all the numbers, we could determine whether it suggests a single scale factor is needed to fix the synthetics or data. So this would be a text-output for debug mode when two sets of seismograms are displayed.

**Comment by bch0w** [comment]
This is now implemented in the `devel` branch with PR #121, can you see if this works as you expect?

RecSec will log out relative amplitude information in the following format:

```
IDX     Y               ID      DIST    AZ      BAZ     TSHIFT  TOFFSET YABSMAX
[2023-08-28 14:13:18] - pysep - DEBUG: 
2       0       NZ.BFZ..BXZ      30.72  120.29  300.08  0.00    0.00    5.93E-03
1       1       NZ.BFZ..BXN      30.72  120.29  300.08  0.00    0.00    2.06E-03
0       2       NZ.BFZ..BXE      30.72  120.29  300.08  0.00    0.00    4.05E-03
2       0       NZ.BFZ..BXZ      30.72  120.29  300.08  0.00    0.00    5.93E-03
1       1       NZ.BFZ..BXN      30.72  120.29  300.08  0.00    0.00    2.06E-03
0       2       NZ.BFZ..BXE      30.72  120.29  300.08  0.00    0.00    4.05E-03

[2023-08-28 14:13:18] - pysep - DEBUG: relative amplitude information
IDX        [O]BS          [S]YN     [O_A]BSMAX  [S_A]BSMAX   O_A/S_A   LN(O_A/S_A)
2      NZ.BFZ..BXZ    NZ.BFZ..BXZ    5.93E-03    5.93E-03    1.00E+00    3.24E-10
1      NZ.BFZ..BXN    NZ.BFZ..BXN    2.06E-03    2.06E-03    1.00E+00    7.48E-10
0      NZ.BFZ..BXE    NZ.BFZ..BXE    4.05E-03    4.05E-03    1.00E+00    5.13E-11
```

At the moment this is providing absmax values and ratios, let me know if you think it would be preferable as standard maximum. 

Also I note that the non-zero ln(o/s) values are from floating point errors because my test data uses the same obs and syn waveforms, which shouldn't be encountered in a real world application.

**Comment by ammcpherson** [comment]
Hi @carltape, @bch0w,

I think that this is working as hoped - I am getting what appears to be a reasonable number (based on visual inspection of the waveform comparisons) for the amplitude ratios.

Thanks,
Amanda

<img width="861" alt="Screen Shot 2023-08-28 at 19 29 23" src="https://github.com/adjtomo/pysep/assets/29667569/55ca67b5-2806-4bdb-93a0-38d04790b963">

<img width="820" alt="Screen Shot 2023-08-28 at 19 25 17" src="https://github.com/adjtomo/pysep/assets/29667569/1092d717-d31c-44d1-9345-e726cd760ea8">
